integration/uinit_test.go

```diff
@@ -22,7 +24,9 @@ func TestHelloWorld(t *testing.T) {
 // TestHelloWorldNegative runs an init which does not print the string "HELLO WORLD".
 func TestHelloWorldNegative(t *testing.T) {
 	// Create the CPIO and start QEMU.
-	tmpDir, q := testWithQEMU(t, "helloworld", []string{})
+	tmpDir, q := testWithQEMU(t, options{
+		uinitName: "helloworld",
+	})
 	defer cleanup(t, tmpDir, q)
```

Comment:
So, ... you have some calls to log.Fatal in places, will this cause any issues if this defer is not run?

Reply:
tbh I lost track of what's running with what.

Reply:
All the Go code inside "integration/testdata/..." is run inside a new process inside of QEMU, so it's safe for it to call `log.Fatal`.

The rest of the code should be calling `t.Fatal`.